name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/wizards/security/LatestQuoteProviderPage.java

```diff
@@ -128,17 +128,17 @@ public void run()
                             valueLatestPrices.setText(Values.Amount.format(p.getValue()));
                             valueLatestTrade.setText(Values.Date.format(p.getTime()));
                             long daysHigh = p.getHigh();
-                            valueDaysHigh.setText(daysHigh == -1 ? Messages.LabelNotAvailable : Values.Amount
```

Comment:
Das sind Änderungen an der Formatierung. Eigentlich habe ich die Formatierungsvorlage eingecheckt und sollte auch verwendet werden. Könntest Du noch mal in Deinen Projekteinstellungen schauen, dass die auch verwendet wird? Deine Formatierung ist besser lesbar (der Zeilenumbruch intelligenter), aber in dem Commit irritiert es etwas. Ich verwende (noch) Luna. Vielleicht kommen die Differenzen auch daher?